engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
@Horneth @kshakir  Was just looking at the scope of this, it looks like `getWorkflow` is being called by WorkflowActor (which currently **does** have the reference to `WorkflowDescriptorMaterializerActor` but **wouldn't** once it is moved from WorkflowManagerSystem to as-a-child-of WorklowManagerActor). Do we want the reference for that actor to be passed in in the constructor to the WorkflowActor ?